pandas/_typing.py

```diff
@@ -25,6 +35,7 @@
 Scalar = Union[str, int, float, bool]
 Axis = Union[str, int]
 Ordered = Optional[bool]
+Serializable = Union[Scalar, Sequence, Mapping]
```

Comment:
Sorry comment might have gotten lost? OK with this move but can you change this to `JSONSerializable`? I think specific that that piece of IO (others may impose their own requirements)

Also I think this should stay with `Dict` and `List` for now; the C extension isn't as flexible as you would hope. Using other types will surface buggy behavior that MyPy can't detect

```python
>>> import pandas._libs.json as json
>>> json.dumps(range(3))
'{"start":0,"stop":3}'  # Probably shouldn't output as a dictionary
>>> json.dumps(UserDict({"a": "b"}))
'{"data":{"a":"b"}}'  # Adds a "data" key the user probably didn't want
```

FWIW the standard module doesn't allow those to be serialized:

```
>>> import json
>>> json.dumps(range(3))
TypeError: Object of type range is not JSON serializable
>>> json.dumps(UserDict({"a": "b"}))
TypeError: Object of type UserDict is not JSON serializable
```

So maybe something can be borrowed from stdlib or typeshed for this alias

Reply:
> So maybe something can be borrowed from stdlib or typeshed for this alias

typeshed just uses `Any`

> Also I think this should stay with `Dict` and `List` for now

agreed.

> OK with this move but can you change this to `JSONSerializable`

sure.